workshops/guided-projects/guided_project_1.ipynb

```diff
@@ -4,14 +4,24 @@
    "cell_type": "markdown",
```

Comment:
<p>Suggestion: I really like that you discuss the unit test files, especially since this is the first time learners are exposed to unit tests in these pipelines. Some additional links to learn more / a quick !file_test.py file exploration with some explanation would be great additions. The default unit tests are quite simple e.g. count feature columns match expected count wouldn't be very useful in practice so another idea is to include an exercise on a future version about adding incrementally adding more useful unit tests e.g. check values generated by tf.Transform within expected standardized [0,1] range, all feature values aren't zero in preprocessing_test.py, no floats or NaNs in an Int feature?</p><p> </p><p>Some good reference resources to link for students here could be:</p><ul><li><a href="https://www.tensorflow.org/community/contribute/tests" target="_blank">https://www.tensorflow.org/community/contribute/tests</a></li><li><a href="https://www.tensorflow.org/api_docs/python/tf/test/TestCase" target="_blank">https://www.tensorflow.org/api_docs/python/tf/test/TestCase</a></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Done!</p>